tests/commands/envelope_test.py

```diff
@@ -170,7 +170,7 @@ class TestSignCommand(unittest.TestCase):
     @staticmethod
     def _make_ui_mock():
         """Create a mock for the ui and envelope and return them."""
-        envelope = mock.Mock()
+        envelope = Envelope()
         envelope.sign = mock.sentinel.default
         envelope.sign_key = mock.sentinel.default
         envelope.headers = {'From': ['foo <foo@example.com>']}
```

Comment:
If you're going to use an actual envelope you should probably set the headers rathaer than replacing them.